tests/schedules_test.py

```diff
@@ -62,7 +62,7 @@ def setUpClass(cls):
             )
 
             # Pick up a random slice and dashboard
-            slce = db.session.query(Slice).all()[0]
+            slce = db.session.query(Slice).filter_by(slice_name="Participants").all()[0]
```

Comment:
[optional] might want to do the same on the next line(s). Also you can use `.one()` instead of `.all()[0]`

Reply:
Since CI in #10485 has passed. I'll do this in another PR.